MapboxDirections/MBRoadClass.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+@objc(MBRoadClass)
+public enum RoadClass: Int, CustomStringConvertible {
+    case toll
```

Comment:
If I understand correctly, the specific `classes` values coming out of the Directions API are dependent on the profile, correct? If there’s a possibility that a different profile might produce exotic values, we should turn this enum into an extensible string enumeration. (`MBDirectionsProfileIdentifier` is an example of an extensible string enumeration.) Otherwise, we’d have to document the fact that unrecognized classes could be dropped entirely.